common/scripts/my-global-command.js

```diff
@@ -0,0 +1,9 @@
+// This script provides the implementation for an example custom command that is
+// defined in common/config/rush/command-line.json.
+
+console.log('Invoking ' + __filename);
```

Comment:
>__filename [](start = 26, length = 10)

?

Reply:
Prints the absolute path of the script file that is executing.

---
In reply to: [217226733](https://github.com/Microsoft/rush-example/pull/2#discussion_r217226733) [](ancestors = 217226733)